From 725d3e212ac7870c8ef9c1ce54cfda156f3e5fee Mon Sep 17 00:00:00 2001
From: SamerKhshiboun <samer.khshiboun@intel.com>
Date: Wed, 2 Nov 2022 10:52:07 +0200
Subject: [PATCH 4/4] disable link time optimization for fw

---
 common/fw/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/common/fw/CMakeLists.txt b/common/fw/CMakeLists.txt
index 5bd018d..4e04505 100644
--- a/common/fw/CMakeLists.txt
+++ b/common/fw/CMakeLists.txt
@@ -44,6 +44,12 @@ set(L53X_FW_URL "${REALSENSE_FIRMWARE_URL}/Releases/L5xx/FW")
 
 add_library(${PROJECT_NAME} STATIC empty.c)
 
+# disable link time optimization for fw by adding -fno-lto to disable -flto flag
+if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  message(STATUS "disable link time optimization for fw by adding -fno-lto to disable -flto flag")
+  target_compile_options(${PROJECT_NAME} PRIVATE -fno-lto)
+endif()
+
 if (MSVC)
   # lib.exe can't handle multiple .res files, so include them in one.
   # even then, the linker won't grab a .res out of a .lib object, so it needs to be explicitly listed
-- 
2.34.1

